app/locale/en.coffee

```diff
@@ -1171,6 +1167,126 @@
     students_enrolled: "students enrolled"
     students_assigned: "students assigned"
     length: "Length:"
+    
+    # New content
```

Comment:
Just so you know, currently [the script]() will produce this output in all the other files:

```diff
-#    length: "Length:"
+#    length: "Length:" # New content
+#    title: "Courses"
```

Better to use that format in the en.coffee file, too, so that the line numbers are the same across languages for reference. Or, if the comment is not timeless (like "New content" here), then we should remove it.